src/eg/src/serialize.rs

```diff
@@ -0,0 +1,53 @@
+use anyhow::{Context, Result};
```

Comment:
Silly nits:

* I like to put the 'use's in near-to-far order. So `use std::...` is first, then crates.io modules like `anyhow`, then local modules, etc. Sometimes `cargo fmt` has other opinions though.

* Can we please name this module `serializable` (serializable.rs) to match the trait names?